src/jvmMain/kotlin/com/sdercolin/vlabeler/repository/ChartRepository.kt

```diff
@@ -231,6 +267,7 @@ object ChartRepository {
     private const val KEY_WAVEFORM = "waveform"
     private const val KEY_SPECTROGRAM = "spectrogram"
     private const val KEY_POWER_GRAPH = "power_graph"
+    private const val KEY_FUNDAMENTAL_GRAPH = "fundamental_graph"
```

Comment:
I feel that we can use simpler ones like `fundamental` and `power`
(If we change the module key, the existing cache is already invalidated, so we can change this key at the same time without extra cost)

Reply:
Done.